<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkMaterialPropertyWidget</class>
 <widget class="QWidget" name="ctkMaterialPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Property</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="ColorLabel">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ctkColorPickerButton" name="ColorPickerButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="displayColorName">
      <bool>false</bool>
     </property>
     <property name="dialogOptions">
      <set>ctkColorPickerButton::UseCTKColorDialog</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="OpacityLabel">
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ctkSliderWidget" name="OpacitySliderSpinBox">
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="AmbientLabel">
     <property name="text">
      <string>Ambient:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkSliderWidget" name="AmbientSliderSpinBox">
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="DiffuseLabel">
     <property name="text">
      <string>Diffuse:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkSliderWidget" name="DiffuseSliderSpinBox">
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="SpecularLabel">
     <property name="text">
      <string>Specular:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ctkSliderWidget" name="SpecularSliderSpinBox">
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="SpecularPowerLabel">
     <property name="toolTip">
      <string>Specular power</string>
     </property>
     <property name="text">
      <string>Power:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ctkSliderWidget" name="SpecularPowerSliderSpinBox">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="BackfaceCullingLabel">
     <property name="text">
      <string>Backface Culling:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="BackfaceCullingCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="PreviewLabel">
     <property name="text">
      <string>Preview:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="ctkMaterialPropertyPreviewLabel" name="MaterialPropertyPreviewLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="PresetsLabel">
       <property name="text">
        <string>Presets:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="PresetsListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>123</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>123</width>
         <height>34</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkMaterialPropertyPreviewLabel</class>
   <extends>QFrame</extends>
   <header>ctkMaterialPropertyPreviewLabel.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>AmbientSliderSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialPropertyPreviewLabel</receiver>
   <slot>setAmbient(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DiffuseSliderSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialPropertyPreviewLabel</receiver>
   <slot>setDiffuse(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpecularSliderSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialPropertyPreviewLabel</receiver>
   <slot>setSpecular(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpecularPowerSliderSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialPropertyPreviewLabel</receiver>
   <slot>setSpecularPower(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorPickerButton</sender>
   <signal>colorChanged(QColor)</signal>
   <receiver>MaterialPropertyPreviewLabel</receiver>
   <slot>setColor(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpacitySliderSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialPropertyPreviewLabel</receiver>
   <slot>setOpacity(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
